test/Drips.t.sol

```diff
@@ -608,91 +611,91 @@ contract DripsTest is Test {
 
     function testSetStreamsRevertsWhenNotCalledByTheDriver() public {
         vm.expectRevert(ERROR_NOT_DRIVER);
-        drips.setStreams(user, erc20, streamsReceivers(), 0, streamsReceivers(), 0, 0);
+        drips.setStreams(account, erc20, streamsReceivers(), 0, streamsReceivers(), 0, 0);
     }
 
     function testGiveRevertsWhenNotCalledByTheDriver() public {
         vm.expectRevert(ERROR_NOT_DRIVER);
-        drips.give(user, 0, erc20, 1);
+        drips.give(account, 0, erc20, 1);
     }
 
     function testSetSplitsRevertsWhenNotCalledByTheDriver() public {
         vm.expectRevert(ERROR_NOT_DRIVER);
-        drips.setSplits(user, splitsReceivers());
+        drips.setSplits(account, splitsReceivers());
     }
 
-    function testEmitUserMetadataRevertsWhenNotCalledByTheDriver() public {
-        UserMetadata[] memory userMetadata = new UserMetadata[](1);
-        userMetadata[0] = UserMetadata("key", "value");
+    function testEmitAccountMetadataRevertsWhenNotCalledByTheDriver() public {
+        AccountMetadata[] memory accountMetadata = new AccountMetadata[](1);
+        accountMetadata[0] = AccountMetadata("key", "value");
         vm.expectRevert(ERROR_NOT_DRIVER);
-        drips.emitUserMetadata(user, userMetadata);
+        drips.emitAccountMetadata(account, accountMetadata);
     }
 
     function testSetStreamsLimitsTotalBalance() public {
         uint128 splitsBalance = uint128(drips.MAX_TOTAL_BALANCE()) / 10;
-        give(user, receiver, splitsBalance);
+        give(account, receiver, splitsBalance);
```

Comment:
In most other places it is called `accountId` but i guess it is fine

Reply:
Fixed